homeassistant/components/auth/__init__.py

```diff
@@ -598,6 +601,50 @@ async def websocket_delete_refresh_token(
     connection.send_result(msg["id"], {})
 
 
+@websocket_api.websocket_command(
+    {
+        vol.Required("type"): "auth/delete_all_refresh_tokens",
+    }
+)
+@websocket_api.ws_require_user()
+@websocket_api.async_response
+async def websocket_delete_all_refresh_tokens(
+    hass: HomeAssistant, connection: websocket_api.ActiveConnection, msg: dict[str, Any]
+) -> None:
+    """Handle delete all refresh tokens request."""
+
+    async def remove_safely(token: RefreshToken) -> bool:
+        # For safety, a revoke_callback could raise an error.
```

Comment:
It's not clear to me what this means.
Does it mean we capture exceptions just in case `async_remove_refresh_token` raises?

Reply:
Yes, a revoke callback could raise an error. See added test